### ISSUE TITLE
Apply long-tap track actions to several tracks

### ISSUE THREAD

**Issue by sonora** [osmandapp/OsmAnd#17061]
https://github.com/osmandapp/OsmAnd/pull/17008 is very cool works great!

One idea to think about: The Samsung file manager is able to apply long-tap  actions not just to one file item but facilitates to select several, via the following UI:
- Upon long-tapping a line item, the logically available actions are not displayed as a popup menu but as a bottom sheet of buttons.
- While this bottom sheet is displayed, the item list is extended by an adfitional selection checkbox column 1 (we could generally move the independent "Visibility" checkbox colomn to the right), which facilitates to add more items to the intended action.
- The bottom sheet is adjusted to always only show the possible actions, e.g. while two or more line items are selected, the "Rename" option disappears.

If we enhanced the UI like this, it would also be possible to e.g. apply appearance changes to _several_ manually selected tracks at once (e.g. even to entire track folders), and could potentially eliminate the need to introduce view groups or similar (related to https://github.com/osmandapp/OsmAnd/issues/16509#issuecomment-1522487746).

**Comment by dmpr0** [comment]
The main goal of the Configure Map – Tracks screen is to change track visibility on the map.

**Comment by vshcherb** [comment]
Long tap should be enabled in My Places menu which could enter Selection mode, similar to File manager but Configure Map is a screen different purpose where selection is utilized to show or not to show on map, so it can't be combined in that case.

**Comment by sonora** [comment]
Yes, I think the checkboxes on that screen should in its regular state simply indicate track visibility on the map.

But I think this may cause no harm, and is actually quite conventional: After a user long-tapped a track, (i.e. when the popup menu is displayed), we temporarily re-dedicate (and facilitate selecting) the checkboxes to indicate for which (potentially several) tracks the user wants to apply the action he is about to perform?
